README.md

```diff
@@ -144,6 +150,56 @@ elements via selectors:
 </simplete-form>
 ```
 
+One customization that we recommend is to add a `<simplete-status>` element
+within the simplete form in order to add information about the status of the
+autocomplete component for assistive technologies. This may not make sense for
+every use case, which is why we have added it as an optional customization.
+Firstly, it is necessary to add the `<simplete-status>` within the
+`<simplete-form>` element:
+
+```
+<form action="/search" method="get">
+    <simplete-form>
+        <label>
+            Search
+            <input type="search" name="query">
+        </label>
+
+        <simplete-status></simplete-status>
+    </simplete-form>
+
+    <button>search</button>
+</form>
+```
+
+When JavaScript is activated, the `<simplete-status>` will become an
+[ARIA live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)
+into which we can inject a status message from the server so that it can be
+announced to assistive technologies. We can then inject our status message by
+adding it as text into an `data-autocomplete-status` attribute within our
+suggestion HTML:
```

Comment:
Could we auto-generate `<simplete-status>` when this attribute is present in suggestions? That would greatly simplify the contract by reducing the number of moving parts authors have to wire up manually.

Reply:
I will investigate if this is possible. I absolutely agree that this would make the contract much better for the developers. The reason I did not do that initially was that I am under the impression that `aria-live` regions only are activated when they are already present in the DOM and will not be announced if they are added later. But I will investigate and see if there is a way to get them to register when adding them dynamically.

Reply:
Just an update here... I don't know what my issue was the last time that I implemented this, but the `search-status` field is definitely announced in both NVDA and VoiceOver even if the `<simplete-status>` is auto generated.

For this reason, I am going to rework this feature so that developers do not have to instantiate the field themselves (if they don't want to).